win32.js

```diff
@@ -44,8 +44,14 @@ module.exports = {
       if (opts.appCopyright) {
         rcOpts['version-string'].LegalCopyright = opts.appCopyright
       }
+      
+      // add "asInvoker" to the built manifest by default
+      rcOpts['requested-executation-level'] = 'asInvoker'
+      if (opts.win32exeLevel) {
```

Comment:
The manifest (http://msdn2.microsoft.com/en-us/library/1w45z383.aspx) changes are not metadata (http://msdn2.microsoft.com/en-us/library/ms404430.aspx) changes, so it would be a bit counter-intuitive to overload that option.

Would a new option, win32manifest.requestedExecutionLevel or similar be more appropriate here ?